staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -108,3 +109,14 @@ if [ "${GENS}" = "all" ] || grep -qw "informer" <<<"${GENS}"; then
            --output-package "${OUTPUT_PKG}/informers" \
            "$@"
 fi
+
+if [ "${GENS}" = "all" ] || grep -qw "openapi" <<<"${GENS}"; then
+  echo "Generating OpenAPI definitions for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/openapi"
+  declare -a OPENAPI_EXTRA_PACKAGES
```

Comment:
why is this needed? we don't appear to be populating it at all

(also arrays in bash is a sign to start considering something other than bash)

Reply:
because it will be needed for apiextensions-apiserver (follow-up PR) which needs autoscaling/v1 as additional group.

Reply:
> also arrays in bash is a sign to start considering something other than bash

It's our choice in k8s.io/code-generator for project not using kubebuilder.

The `declare -a` makes sure that `${OPENAPI_EXTRA_PACKAGES[@]}` in the next line does not fail when the variable is empty.

Reply:
Yeah, I understand this is the language you're locked into, and it's a sane/clean declaration given that constraint. I just wanted to make sure this wasn't something being left around just incase, follow on makes sense.